src/hw12_bookparser/BookReader.java

```diff
@@ -0,0 +1,32 @@
+package hw12_bookparser;
+
+import java.io.*;
+import java.nio.file.Path;
+import java.util.*;
+
+public class BookReader{
+    private static final Path DIRECTORY_PATH = Path.of("src/hw12_bookparser/books");
+    // Method for reading the file and getting content
+    public static String getBookContent(String bookTitle) throws IOException {
```

Comment:
It's better to handle exceptions as close as possible to the place they may occure